css-pseudo-classes/styles.css

```diff
@@ -0,0 +1,21 @@
+button {
+  box-sizing: border-box;
+  font-size: 23px;
+  margin: auto;
+  padding: 15px 40px;
+}
+
+button:hover {
+  cursor: pointer;
+  color: white;
+  background-color: black;
+}
+
+.gray {
+  background-color: lightgray;
+}
```

Comment:
Try accomplishing this using the `:nth-child()` psuedo-class rather than giving the elements a class. Technically both of these options are valid and will work, but since this excercise revolves around psuedo-classes and understanding them, I'd like you to learn how to use them.

Reply:
Ahh okay gotcha